tests/data/tensorflow-serving-response-data.json

```
@@ -0,0 +1 @@
+{"data": {"names": ["t:0", "t:1", "t:2", "t:3", "t:4", "t:5", "t:6", "t:7", "t:8", "t:9"], "tensor": {"shape": [1, 10], "values": [8.49343338e-22, 2.85119398e-35, 0.123584226, 0.0665731356, 1.18265652e-28, 0.809836566, 4.16546084e-13, 1.48641526e-19, 6.06191043e-06, 2.40174282e-20]}},"meta": {"requestPath": {"classifier": "IMAGE:VERSION"}}}
```

Let's add a new line if we end up keeping this.
```suggestion
{"data": {"names": ["t:0", "t:1", "t:2", "t:3", "t:4", "t:5", "t:6", "t:7", "t:8", "t:9"], "tensor": {"shape": [1, 10], "values": [8.49343338e-22, 2.85119398e-35, 0.123584226, 0.0665731356, 1.18265652e-28, 0.809836566, 4.16546084e-13, 1.48641526e-19, 6.06191043e-06, 2.40174282e-20]}},"meta": {"requestPath": {"classifier": "IMAGE:VERSION"}}}

```

---

Will do.